specimen/wildlife-sounds-soundscapes/eml.xml

```diff
@@ -1,10 +1,7 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<eml:eml xmlns:eml="eml://ecoinformatics.org/eml-2.1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-packageId="XXXXXXXXX" scope="system" system="http://gbif.org" xml:lang="en"
-xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 http://rs.gbif.org/schema/eml-gbif-profile/1.1/eml.xsd">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><eml:eml xmlns:eml="eml://ecoinformatics.org/eml-2.1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" packageId="b1047888-ae52-4179-9dd5-5448ea342a24" scope="system" system="http://gbif.org" xml:lang="en" xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 http://rs.gbif.org/schema/eml-gbif-profile/1.1/eml.xsd">
     <dataset>
         <alternateIdentifier/>
-        <title>Xeno-canto - Soundscapes from around the world</title>
+        <title>Xeno-canto - Chiroptera sounds from around the world</title>
```

Comment:
Hoi @jeroencreuwels, hier is wat fout gegaan denk ik. Dit is de eml van de soundscapes, niet de van de vleermuizen. Zou je dat kunnen aanpassen?

Reply:
er is inderdaad iets verwisseld tussen wildlife sounds en bats

Reply:
Heb dit aangepast en getest